### PR TITLE
Business newsletter subscriber

### PR DESCRIPTION
**Pull request by emmabckstrm**
## Business newsletter

- Added a sign up section in the bottom of each business page
- Removed the regular newsletter sign up from business pages
- The sign up only accepts valid and non-duplicated emails (duplicate emails are still shown as "accepted" to the user)
- When signing up, a confirmation email is sent to the address, see below for content.

Screenshot 2021-09-13 at 10 55 56

Screenshot 2021-09-13 at 10 56 41

Screenshot 2021-09-13 at 10 55 25

---

## Regular newsletter

- Did some general refactoring of the code
- Added sign up confirmation email, see content below

Screenshot 2021-09-13 at 10 55 39

### CODE REVIEW

File: app/controllers/business_newsletter_subscribers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class BusinessNewsletterSubscribersController < ApplicationController
+  def create
```
[the remark this answers is not in the document]
Comment (emmabckstrm) [reply app/controllers/business_newsletter_subscribers_controller.rb:4]: Here I'm not really sure what to do. I can see this going either way. What I've done now is two separate tables for the two different subscribers, with different models and controllers and all. But I guess it technically could be one model with a `type` attribute for business or regular emails. 

What are your guys' thoughts?

**Comment by emmabckstrm** [comment]
[Sign up for business newsletter](https://trello.com/c/KOcbkcq7/670-sign-up-for-business-newsletter)

**Comment by emmabckstrm** [comment]
@nanditagoclimate @stefanlindbohm I've left the codeclimate error to have your input on it :) so the PR is ready for reviews

File: app/controllers/business_newsletter_subscribers_controller.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (nanditagoclimate) [reply app/controllers/business_newsletter_subscribers_controller.rb:4]: Yes, +1 on that. Let's introduce a type in NewsletterSubscriber and reuse the same controller/model and even specs.

**Comment by emmabckstrm** [comment]
Closing PR, waiting with removing the branch as I no longer have a local copy of it

**Pull request closed without merging**